### ISSUE TITLE
Malfunctioning license selection in CKAN 2.9

### ISSUE THREAD

**Issue by nfee006** [asc-csa/ckanext-asc-csa#123]
License selection appears as an empty dropdown

![image](https://user-images.githubusercontent.com/11013061/120343812-069a7b80-c2c7-11eb-83ec-a7d0d678a772.png)
 
Any luck with this @coleweusthem?
This might be useful info: https://github.com/ckan/ckanext-scheming/issues/207

**Comment by coleweusthem** [comment]
@nfee006, no luck yet, works if I turn off the CSA extensions though, so I'm looking at the ckan version of scheming to see if they are doing something different

**Comment by nfee006** [comment]
Can't tell if this is relevant but just in case: https://github.com/ckan/ckanext-scheming/issues/277

**Comment by coleweusthem** [comment]
@nfee006, I've got licenses showing but there are only 2 showing from the list, it seems like there is some kind of filter. Is this intentional? 
<img width="615" alt="Screen Shot 2021-06-01 at 2 26 53 PM" src="https://user-images.githubusercontent.com/46227765/120365935-6acc3800-c2e5-11eb-94aa-a6210715fe3f.png">

**Comment by nfee006** [comment]
Great! Yeah, I saw the filter too. I'm sending an email to the person who manages the CSA metadata/licenses to check. 

What was the solution by the way?

**Comment by coleweusthem** [comment]
I had to replace most of the plugin with the latest code from the CKAN version

**Comment by nfee006** [comment]
I just pulled your latest edits on the python 3 branch - fixed the license issue on my side as well. I did notice a new scheming issue but I'll make a new issue.

I'm still waiting to hear back regarding the license filter but I think we can leave it as-is for now. I just tested it and I'm easily able to remove the filter if need be (for my future reference, that edit must be applied to the presets.json file within ckanext-asc-csa and not the one in scheming).
